cli_widgets/hidden_input_guard.hpp

```diff
@@ -48,6 +48,13 @@ namespace cli_widgets {
   
   struct hidden_input_guard {
     hidden_input_guard(bool enable = true) { set_stdin_echo(enable); }
+
+    inline std::string ask_for_passphrase(){ 
```

Comment:
I just don't like the naming of this because it implies from an API perspective that any hidden input is a "passphrase"